packages/ethereum-contracts/contracts/superfluid/Superfluid.sol

```diff
@@ -861,6 +877,18 @@ contract Superfluid is
                     valueForwarded ? 0 : msg.value,
                     operations[i].data);
                 valueForwarded = true;
+            } else if (operationType == BatchOperation.OPERATION_TYPE_SIMPLE_FORWARD_CALL) {
+                (bool success, bytes memory returnData) =
+                    DMZ_FORWARDER.forwardCall(operations[i].target, operations[i].data);
```

Comment:
should also forward msg.value

Reply:
Can't use `msg.value` because it reflects the initial value, not the current one.
But we can use `address(this).balance` instead since the contract doesn't hold any ETH.